### ISSUE TITLE
trainable=True, trainable params doesn't change

### ISSUE THREAD

**Issue by konstantin-frolov** [keras-team/keras#13587]
Hi people. I have some problems with transfer learning. I use the facenet model and load the model and weights from * .h5 files.
I want to transfer learning of this model for my classifier. Then I change the weight value of some layers to True. I change it, but after compiling a new model with my classifier, the number of trained parameters does not change, all loaded models weights in an untrained state.
`Layer (type)                 Output Shape              Param #   `
`inception_resnet_v1 (Model)  (None, 128)               22808144  `
`dense_1 (Dense)                     (None, 256)               33024     `
`activation_50 (Activation)   (None, 256)               0         `
`dropout_1 (Dropout)          (None, 256)               0         `
`dense_2 (Dense)              (None, 3)                 771       `
`activation_51 (Activation)   (None, 3)                 0         `

`Total params: 22,841,939`
`Trainable params: 33,795`
`Non-trainable params: 22,808,144`

Thank you!

**Comment by konstantin-frolov** [comment]
I just needed to set model.trainable = True
